### DISCUSSION TITLE
Better Error Messages from Flows

### DISCUSSION THREAD

**Discussion by jakedowns** [directus/directus#16288] [Feature Requests]
![image](https://user-images.githubusercontent.com/1683122/199605355-40e61c84-cba4-4efa-9c6a-4a2e8c464179.png)

it'd be nice to know which flow block was throwing this error. currently i have to disconnect blocks until i find the offending one.

**Comment by azrikahar** [comment]
Would you mind sharing the operation that you had narrowed down which causes this error in your setup? Could it be "Run Script", or something else?

**Reply by jakedowns** [reply]
i should've been more specific. i don't recall.

it must've been either a payload field on a Update Data op, or a Log to Console op.

if i run across it again i'll post it here

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
